wingtips-zipkin/README.md

```diff
@@ -26,9 +26,22 @@ Parameters for this basic `WingtipsToZipkinLifecycleListener` constructor:
 * **`localComponentNamespace`** - The `zipkin.Constants.LOCAL_COMPONENT` namespace that will be used when creating certain Zipkin annotations when the Wingtips span's `Span.getSpanPurpose()` is `LOCAL_ONLY`. See the `zipkin.Constants.LOCAL_COMPONENT` javadocs for more information on what this is and how it's used by the Zipkin server, so you know what value you should send.
 * **`postZipkinSpansBaseUrl`** - The base URL of the Zipkin server. This should include the scheme, host, and port (if non-standard for the scheme). e.g. `http://localhost:9411`, or `https://zipkinserver.doesnotexist.com/`.
 
-## Using Native Zipkin Brave `SpanCollector`s
+## Using Native Zipkin `Reporter`s
+
+The default `ZipkinSpanSender` implementation provided by this submodule (used under the hood when you follow the Quickstart instructions above) for sending spans to the Zipkin server uses `URLConnectionReporter` from the official Zipkin `zipkin-reporter-urlconnection` library. This should work decently for many use cases, however it is limited to HTTP transport. You can trivially use any of the other reporters provided by the [zipkin-reporter-java](https://github.com/openzipkin/zipkin-reporter-java) repository by pulling in the dependencies necessary to have access to the `Reporter` implementation you want to use, and then initialize with the following:
```

Comment:
ahh well if documenting this, than shade/shadow won't work :) I might suggest holding off on merging this until monday, as I'm refactoring some of the key problems in the current design. Most important is kafka has a blocking api problem, which means the "Sender" design I'm working on will include a health check. At bootstrap, folks can invoke that to make sure kafka is available as opposed to having potentially a minute of blocking then an exception!